src/gui/reports/ReportsDialog.cpp

```diff
@@ -113,16 +120,33 @@ void ReportsDialog::reject()
     emit editFinished(true);
 }
 
-void ReportsDialog::entryActivationSignalReceived(const Group* group, Entry* entry)
+void ReportsDialog::entryActivationSignalReceived(QWidget* sender, const Group* group, Entry* entry)
 {
+    m_sender = sender;
```

Comment:
This is what `Object::sender()` is for. You should not be passing the sender in the signal.